oracle.py

```diff
@@ -208,5 +297,64 @@ def check(learned_side_effect, learned_status, testing_side_effect, testing_stat
             bug_report += "[DEBUGGING SUGGESTION] %s\n" % generate_debug_suggestion(
                 testing_config)
         bug_report = "\n[BUG REPORT]\n" + bug_report
+
+        if learned_resources != None:
```

Comment:
@laphets 
One last question before merge: it seems that we still use status.json rather than resource.json to decide whether there is a bug or not, is that correct?

Reply:
If there is `resource.json` generated from learning, the new oracle will take into effect.

---

For now, I only generate `resource.json` for certain workloads. In the future, after the new oracle feature is more stable, we can run a batch and generate `resource.json` for all our current workloads.

Reply:
Sounds good. Let's merge this giant now.